### DISCUSSION TITLE
Making Pressable onHoverIn Typescript happy?

### DISCUSSION THREAD

**Discussion by jysung** [necolas/react-native-web#2107] [Help] [answered]
I use `react-native-web` as an alias of `react-native`.
This web package has additional props like `onHoverIn` & `onHoverOut` on `Pressable`, does anyone have any idea how I would make Typescript happy? Thanks!

```
  Property 'onHoverIn' does not exist on type 'IntrinsicAttributes & PressableProps & { children?: ReactNode; }'.ts(2322)
```

**Comment by shawngustaw** [comment, marked as answer]
You can use [declaration merging](https://www.typescriptlang.org/docs/handbook/declaration-merging.html) to annotate web specific properties you're using, i.e.:

```
// overrides.d.ts
import type { MouseEvent } from 'react';

declare module 'react-native' {
  interface TouchableWithoutFeedbackProps {
    onHoverIn?: (event: MouseEvent<HTMLElement>) => void;
    ...
  }
}
```

This has been working for me. Hope it helps!
